docs/index.asciidoc

```diff
@@ -157,7 +180,17 @@ guaranteed to work correctly with the AWS SDK.
   * Value type is <<string,string>>
   * Default value is `nil`
 
-Ruby style regexp of keys to exclude from the bucket
+Ruby style regexp of keys to exclude from the bucket.
```

Comment:
Perhaps worth noting, excluding files here does _not_ save us the cost of listing the files, since they are excluded between obtaining their listing and processing them.

```suggestion
Ruby style regexp of keys to exclude from the bucket.
Files matching the pattern are skipped _after_ they have been listed, so it is recommended to use <<plugins-{type}s-{plugin}-prefix>> instead where possible.
```